scripts/Tools/load.awk

```diff
@@ -1,40 +0,0 @@
-BEGIN {
```

Comment:
It's easy to add something back if needed, I've already put out a call far and wide for users to speak up, i don't want to do it again.